Tests/Marketplace/upload_packs.py

```diff
@@ -570,8 +565,65 @@ def add_private_packs_to_index(index_folder_path: str, private_index_path: str):
             update_index_folder(index_folder_path, d.name, d.path)
 
 
+def was_private_pack_updated(private_index_json, public_index_json):
+    """
+    Checks if Content commit hash of each private pack in private and public index files are equal. If there is at
+        least one private pack that has a different content commit hash, it tells us that this pack was updated and
+        should be uploaded. So, an upload flow should NOT be skipped.
```

Comment:
IMO this function is redundant, let's combine it with the `is_private_packs_updated` function.